staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_v2.go

```diff
@@ -0,0 +1,455 @@
+/*
```

Comment:
If the intent is to provide an alternative to `store.go`, maybe make changes directly to that file so the proposed changes can more easily be reviewed in context?

Reply:
I agree for diff viewing but this approach was suggested by the folks with a lot of rebases under their belt as being easier to maintain /shrug